src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,75 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import imageUrl from "../../static/images/card/card.png";
+import * as styles from "./Card.module.css";
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+interface CardContentProps {
+  header: string;
+  para: string;
+}
+
+export const CardImage = () => {
+  return <img src={imageUrl} alt="" />;
+};
+
+export const CardContent = ({
+  header = "Card Name",
+  para = "Lorem ipsum dolor sit amet consectetur, adipisicing elit. Enim harum\n" +
```

Comment:
**nitpick:** We should not set such default value - we don't need these props anyway